content/02.introduction.md

```diff
@@ -69,7 +69,7 @@ Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1
 In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
 CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
 These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
-In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including the _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and the _Middle East respiratory syndrome coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
+In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
```

Comment:
The _mSystems_ guidelines ask authors to use the [ICTV taxonomy](https://talk.ictvonline.org/taxonomy/) -- they don't have SARS-CoV-2 in there yet, but I just assume that it will be similar to SARS-CoV when they do add it!